package-lock.json

```diff
@@ -14,6 +14,9 @@
         "shared/*",
         "docs/examples/*"
       ],
+      "dependencies": {
+        "@types/node": "^20.5.9"
+      },
```

Comment:
Just a reminder that these lines creep in for optional dependencies

Not sure why, but running `npm install` on the Dependabot PR fixes it